tests/fixtures/page-router/netlify/functions/purge-cdn.ts

```diff
@@ -0,0 +1,41 @@
+import { purgeCache, Config } from '@netlify/functions'
+
+export default async function handler(request: Request) {
+  const url = new URL(request.url)
+  const pathToPurge = url.searchParams.get('path')
+
+  if (!pathToPurge) {
+    return Response.json(
+      {
+        status: 'error',
+        error: 'missing "path" query parameter',
+      },
+      { status: 400 },
+    )
+  }
+  try {
+    await purgeCache({ tags: [`_N_T_${encodeURI(pathToPurge)}`] })
+    return Response.json(
+      {
+        status: 'ok',
+      },
+      {
+        status: 200,
+      },
+    )
+  } catch (error) {
+    return Response.json(
+      {
+        status: 'error',
+        error: error.toString(),
+      },
+      {
+        status: 500,
+      },
+    )
+  }
+}
+
+export const config: Config = {
+  path: '/api/purge-cdn',
+}
```

Comment:
I had some troubles with this function being in Next.js api handler - in the end I don't rely on it for added test, but overall it doesn't really make sense to use Next.js API for it, if it can use just vanilla Netlify function, as it only purge cdn cache, and not any Next.js caches in blobs